drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -218,6 +218,12 @@ class Hmis::Form::Definition < ::GrdaWarehouseBase
   }.freeze
   NON_QUESTION_ITEM_TYPES = ['DISPLAY', 'GROUP'].freeze
 
+  # NON_ADMIN_FORM_ROLES are editable by everyone with can_manage_forms permission
+  NON_ADMIN_FORM_ROLES = [
```

Comment:
This naming has been tripping me up a little bit. I think it might be clearer to just name it in the way you did in the code comment. `USER_EDITABLE_FORM_ROLES`? I think the confusing part is that the people editing them are still "admins", and the var name doesn't mention "editable" or "configurable".

Reply:
Ah, yeah I struggled a lot with this naming. I had `EDITABLE_FORM_ROLES` initially and would be fine with changing it back to something similar. The reason I moved away from that is that I started to confuse myself about this group of form rules being non-editable for some users. For example, in the [`form_definition`](https://github.com/greenriver/hmis-warehouse/blob/4c4a9e76d153de126451670502e2db9b30a60b10/drivers/hmis/app/graphql/types/hmis_schema/query_type.rb#L385-L395) and [`form_identifier(s)`](https://github.com/greenriver/hmis-warehouse/blob/4c4a9e76d153de126451670502e2db9b30a60b10/drivers/hmis/app/graphql/types/hmis_schema/query_type.rb#L406-L424) queries, this list is used to determine which models are available to view for users who have `can_configure_data_collection` but not `can_administer_config` -- but they aren't necessarily able to edit, which is determined by `can_manage_forms`.

Reply:
Ah right, we're gating visibility on them too, right now. I guess NON_ADMIN_FORM_ROLES makes sense then!